src/server/connection.cpp

```diff
@@ -41,13 +41,13 @@ void Connection::handle_read(const boost::system::error_code &error, std::size_t
 
     // no error detected, let's parse the request
     http::compression_type compression_type(http::no_compression);
-    util::tribool result;
+    RequestStatus result;
```

Comment:
This needs to be `RequestParser::RequestStatus`. Remember this is a nested type and we are outside of `RequestParser`.